diff --git a/Formula/node.rb b/Formula/node.rb
index 3c420710591..f7dc99265d5 100644
--- a/Formula/node.rb
+++ b/Formula/node.rb
@@ -92,7 +92,8 @@ class Node < Formula
     # Enabling LTO errors on Linux with:
     # terminate called after throwing an instance of 'std::out_of_range'
     # LTO is unpleasant if you have to build from source.
-    args << "--enable-lto" if OS.mac? && build.bottle?
+    # NOTE: LTO disabled on Mojave because it breaks bottle builds
+    # args << "--enable-lto" if OS.mac? && build.bottle?
 
     system "./configure", *args
     system "make", "install"
